packages/storybook8/stories/Concepts/RealTimeText/Docs.mdx

```diff
@@ -89,3 +67,27 @@ Note that Real-Time Text will be enabled for all participants in the call once t
 
 To read more details about the underlying infrastructure, click here:  
 [Real Time Text (RTT) Overview - An Azure Communication Services concept document | Microsoft Learn](https://learn.microsoft.com/en-us/azure/communication-services/concepts/voice-video-calling/real-time-text)
+
+## FAQ
+
+### How can I enable RTT?
+
+RTT can be enabled during calls via the control bar “more” button.
+
+Enabling RTT opens the RTT window for everyone in the call. When one user enables RTT, other users do not need to enable it for themselves. This window cannot be closed for the duration of the call nor can RTT be disabled.
+
+### What versions of UI Library support RTT?
+
+RTT is currently in public preview and supported in 1.24.0-beta.1.
```

Comment:
is it better to just have a banner on top indicating that it's in public preview instead of having a section here indicating a version #?

Reply:
Updated